syntax = "proto3";

package example;

option cc_generic_services = true;

message RequestData {
    string data = 1;
}

message ResponseData {
    string data = 1;
}

service ExampleService {
    rpc Echo (RequestData) returns (ResponseData);
}
